python-modules/cis_publisher/cis_publisher/publisher.py

```diff
@@ -151,16 +151,19 @@ def post_all(self, user_ids=None):
         return ret
 
     def _really_post(self, user_id, qs, profile, failed_users):
-        response_ok = False
-        retries = 0
-        access_token = self._get_authzero_token()
-
         # Existing users (i.e. users to update) have to be passed as argument
         if user_id in self.known_cis_users_by_user_id:
             qs = "/v2/user?user_id={}".format(user_id)
         # New users do not
         else:
             qs = "/v2/user"
+        self._really_post_with_qs(user_id, qs, profile, failed_users)
```

Comment:
can't this just replace `_really_post` instead?

Reply:
I'd rather do that as follow up to make it easily revertible. We're not using `qs` in `_really_post, but we're passing it in several cases. What `_really_post_with_qs` does not do is the implicit checking for known users. I'd propose I refactor this in follow up and have a `_really_post_with_known_userid` and a `_really_post` method or something like that.

Reply:
yeah sounds better